src/components/Renderer/index.tsx

```diff
@@ -25,14 +25,13 @@ export const getValue = (
 	schema?: JSONSchema,
 	uiSchema?: UiSchema,
 ) => {
-	const calculatedValue = get(uiSchema, 'ui:value', value);
+	const calculatedValue = uiSchema?.['ui:value'] ?? value;
 	// Fall back to schema's default value if value is undefined
-	return calculatedValue !== undefined
-		? calculatedValue
-		: get(schema, 'default');
+	return calculatedValue ?? schema?.default;
```

Comment:
From the comment it looks like the `undefined` only check might have been intentional :thinking: 
@grahammcculloch ^ ?

Reply:
@thgreasi yes exactly. `null` is a valid value in this case so we want to explicitly test for `undefined`.

Reply:
Urg, you're right. I saw so many cases around the codebase where it's pre `??` that I assumed this was one as well. I've fixed it so explicit `null` vs `undefined` in `ui:value` will also be respected